doc/tutorials/gapi/face_beautification/face_beautification.markdown

```diff
@@ -0,0 +1,500 @@
+# Implementing a face beautification algorithm on G-API {#tutorial_gapi_face_beautification}
+
+@prev_tutorial{tutorial_gapi_anisotropic_segmentation}
+
+[TOC]
+
+# Introduction {#gapi_fb_intro}
+
+In this tutorial you will learn:
+* How to implement a custom kernel (using OpenCV backend);
+* How to use Inference-API to infer networks inside the graph;
+* How to process a videostream inside the graph using Streaming-API;
+* The main ideas of the Face Beautification algorithm.
+
+You can find source code in the `modules/gapi/samples/face_beautification.cpp`
```

Comment:
This file shouldn't be there. Please move it to `samples/cpp/tutorial_code/gapi/`